meson.build

```diff
@@ -463,6 +463,42 @@ endif
 
 
 
+###############################################################
+# Library: Tcl (for pltcl)
+
+# tclConfig.sh
+#
+# NB: tclConfig.sh is used in autoconf build for getting
+# TCL_SHARED_BUILD, TCL_INCLUDE_SPEC, TCL_LIBS and TCL_LIB_SPEC
+# variables. That is not implemented in meson build.
+###############################################################
```

Comment:
Good!

Small improvement: I'd say "For now we have not seen a need to copy that behaviour to the meson build" instead of the "That is not implemented" bit.